Makefile

```diff
@@ -87,5 +87,5 @@ test-snowflake-only:
 	cd kedro-datasets && pytest --no-cov --numprocesses 1 --dist loadfile -m snowflake
 	cd kedro-datasets && pytest kedro_datasets/snowflake --doctest-modules --doctest-continue-on-failure --no-cov
 
-rtd:
-	cd kedro-datasets && python -m sphinx -WETan -j auto -D language=en -b linkcheck -d _build/doctrees docs/source _build/linkcheck
+check-docs:
+	cd $(plugin) && python -m sphinx -WETan -j auto -D language=en -b linkcheck -d _build/doctrees docs/source _build/linkcheck
```

Comment:
Not really 😅 Since others don't have docs. I was hesitating between this, or calling the action `check-datasets-docs`. Wdyt?

And of course, Kedro goes next 🔥

Reply:
I'd prefer to keep it specific so it's clear this command just works for datasets. `check-datasets-docs` sounds good.

Reply:
Done!